### PR TITLE
DOC: Corrected broken usage link

### LINKED ISSUE #3: [DOC]: Broken usage link
# Incorrect Documentation

## Describe the error

The usage instructions linked here-  https://github.com/nhcarrigan/discord-rpc#creating-your-own
link to USAGE.MD.. but USAGE.md is the file in the repo.
## Expected information

To see how to set up your own RPC, view our [usage instructions](https://github.com/nhcarrigan/discord-rpc/blob/main/USAGE.md).

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
# Pull Request

## Description:

 

Corrected broken usage link in README.md.

## Related Issue:

Closes #3 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ You can view this by looking at my presence on Discord!
 
 ## Creating Your Own
 
-To see how to set up your own RPC, view our [usage instructions](./USAGE.MD).
+To see how to set up your own RPC, view our [usage instructions](https://github.com/nhcarrigan/discord-rpc#creating-your-own).
```
Comment (nhcarrigan) [inline README.md:14]:
```suggestion
To see how to set up your own RPC, view our [usage instructions](./USAGE.md).
```
Linking to the same section of the README doesn't accomplish much. 🙂 

As discussed in the issue, we should only need to change the casing of the file extension to resolve the broken link.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (96RadhikaJadhav) [reply README.md]: Done :) .

**Review by nhcarrigan — APPROVED** [review]
Looks good to me.

**Comment by nhcarrigan** [comment]
Congrats on your first pull request (PR).

Thank you for your contribution to the page. We are happy to accept these changes and look forward to future contributions.

**Comment by nhcarrigan** [comment]
@all-contributors please add @96RadhikaJadhav for docs

**Pull request merged**